spec/lrama/option_parser_spec.rb

```diff
@@ -343,4 +343,24 @@
       end
     end
   end
+
+  describe "@define" do
```

Comment:
Could you add a test case for the issue which this PR will resolve, in short test case for `--define=foo`?

Reply:
Added: https://github.com/ruby/lrama/pull/527/commits/5dfb6a3f77e7877a3dc962bcb05efd6c50889b11